doc/hintsntips_luna.rst

```diff
@@ -292,57 +292,57 @@ To make node self-contained we still need to add bootloader and change fstab to
     umount /sysroot/proc
 
 
-To edit script simply run::
+To edit the script, simply run::
 
     # luna group change new-compute --partscript --edit
 
-It will open editor for you. In addition it supports piping::
+It will open the editor. In addition, it supports piping::
 
     # cat compute-part.txt | luna group change compute --partscript --edit
 
 Other configurable items in Luna
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Switches are required to be configured if you want to make Luna automatically discover nodes' MAC addresses. It is crucial to check if switch provides information about learned MAC addresses vin SNMP::
+Switches must be configured for Luna to automatically discover nodes' MAC addresses. It is crucial to check if a switch provides information about learned MAC addresses vin SNMP::
 
     # snmpwalk -On -c public -v 1 SWITCH_IP .1.3.6.1.2.1.17.7.1.2.2.1
 
 It should list something like::
 
     .1.3.6.1.2.1.17.7.1.2.2.1.2.1.24.102.218.96.27.201 = INTEGER: 210
 
-Last 6 numbers is some MAC address in decimal format. See ``man luna`` for more information how to decrypt it.
+The last 6 numbers is a MAC address in decimal format. See ``man luna`` for more information on how to decrypt it.
 
-When Luna is able to get MAC addresses from switches it can display them in ``luna cluster listmacs``.
```

Comment:
...it ~~can~~ will display them in ``luna cluster listmacs``.
I'd use a more affirmative tone